packages/forms/src/widgets/DatalistWidget/DatalistWidget.js

```diff
@@ -151,12 +151,16 @@ class DatalistWidget extends React.Component {
 		};
 	}
 
-	onBlur() {
+	onBlur(event) {
 		if (this.props.options &&
-			this.props.options.restricted && this.state.initalItems.indexOf(this.state.value) === -1) {
+			this.props.options.restricted &&
+			this.state.initalItems.indexOf(this.state.value) === -1) {
```

Comment:
`Array.prototype.includes()` can be used here :

Instead of :
`this.state.initalItems.indexOf(this.state.value) === -1`

You can write :
`!this.state.initalItems.includes(this.state.value)`